edit/tty/reader.go

```diff
@@ -196,7 +203,7 @@ func (rd *Reader) readOne(r rune) {
 				}
 				mod := mouseModify(int(cb))
 				unit = MouseEvent{
-					Pos{int(cy) - 32, int(cx) - 32}, down, button, mod}
+					Pos{int(cy) - 32, int(cx) - 32}, down, button, mod | prefixMod}
```

Comment:
Mouse events have very different escape sequences. Do you have evidence that double ESC can arise in this case? If not, I'd rather you remove this.

Reply:
Ok, i have just removed this.